docs/pages/1-generate-https.md

```diff
@@ -168,205 +166,64 @@ this is pronounced ENGINE-X - it took me only 8 years to know that :P). As a san
-DNS.1 = $DOMAIN
-DNS.2 = www.$DOMAIN
-EOF
-```
-
-This should produce a file `csr_details.txt` in your present working directory
-that we will use later.
-
-
-### Step 2. Install Acme Tiny
-We are going to use [Acme Tiny](https://github.com/diafygi/acme-tiny) to both issue
-and renvew our certificates. Let's install it first. I chose to clone to `/tmp` and
-install to `/opt`, you can obviously mix this up. If you are using a Droplet (or
-a server without python) make sure to install it first:
+Certbot automates certificate generation and renewal. In other words, it makes it really easy to setup SSL.
 
-```bash
-# Python
-sudo apt-get install -y python
 ```
-
-Then install Acme Tiny.
-
-```bash
-sudo mkdir /opt/acme_tiny
-cd /tmp && git clone https://github.com/diafygi/acme-tiny
-sudo mv acme-tiny /opt/acme-tiny/
-sudo chown $USER -R /opt/acme-tiny
+sudo add-apt-repository ppa:certbot/certbot
+sudo apt-get update
+sudo apt-get install python-certbot-nginx
 ```
 
-### Step 3. Create Account Key and Parameters
-This step you only need to do once, so I'm bundling it into one. You need to generate a 
-key for your server, and for the best encryption, a `dhparam.pem` key (it takes a while!).
+### Step 2. Get certificates with certbot
 
-```bash
-# Generate a private account key, if doesn't exist
-if [ ! -f "/etc/ssl/certs/account.key" ]
-   then
-   openssl genrsa 4096 > account.key && sudo mv account.key /etc/ssl/certs
-fi
+Now obtain a certificate by running this command. 
+You can specify multiple hostnames, such as one with "www" like this:
 
-# Add extra security (takes a while!)
-if [ ! -f "/etc/ssl/certs/dhparam.pem" ]
-   then
-   openssl dhparam -out dhparam.pem 4096 && sudo mv dhparam.pem /etc/ssl/certs
-fi
 ```
-
-You shouldn't need to regenerate these files.
-
-### Step 4. Backup Previous Key / Certificate
-This step you only need to do if you have a previously created certificate and domain
-key. It's probably not super necessary, but I do it anyway. The certificate we will make
-(and/or backup) is called `expfactory.cert` and the key is `expfactory.key`.
-
-```bash
-# backup old key and cert
-if [ -f "/etc/ssl/private/domain.key" ]
-   then
-   sudo cp /etc/ssl/private/domain.key{,.bak.$(date +%s)}
-fi
-
-if [ -f "/etc/ssl/certs/chained.pem" ]
-   then
-   sudo cp /etc/ssl/certs/chained.pem{,.bak.$(date +%s)}
-fi
-
-if [ -f "/etc/ssl/certs/domain.csr" ]
-   then
-   sudo cp /etc/ssl/certs/domain.csr{,.bak.$(date +%s)}
-fi
+certbot certonly --nginx -d expfactory.dynu.net -d www.expfactory.dynu.net
 ```
 
-### Step 5. Call Openssl
-We now are going to use openssl with our `csr_details.txt` to generate a new domain key! That looks like this:
+This will save all of the certificate files to /etc/letsencrypt/live/expfactory.dynu.net (the first domain listed in the cerbot command). It will ask for your email during the generation. This is used to send you renewal reminders at 20 days, 10 days, and 1 day before expiry (super helpful!)
 
```

Comment:
oh hmm, so you are saying the above command generates a certificate for each domain? Is that a better approach than having one?

Reply:
It does, but it combines them into a single certificate file using the command above.

Reply:
ah ok, so the user only sees one file. That works!